src/Platforms/SYCL/SYCLDeviceManager.cpp

```diff
@@ -130,6 +130,10 @@ SYCLDeviceManager::SYCLDeviceManager(int& default_device_num, int& num_devices,
     default_device_queue = std::make_unique<sycl::queue>(visible_devices[sycl_default_device_num].get_context(),
                                                          visible_devices[sycl_default_device_num].get_device(),
                                                          sycl::property::queue::in_order());
+    if (!visible_devices[sycl_default_device_num].get_device().has(sycl::aspect::ext_intel_free_memory))
+      app_warning()
+          << "Free memory queries always return 0 due to inactive 'oneAPI' System Resource Management (sysman). "
+          << "Set environment variable ZES_ENABLE_SYSMAN to 1 to activate the query feature." << std::endl;
```

Comment:
@prckent if `ZES_ENABLE_SYSMAN` not set, users get the warning above. I explored ways of setting `ZES_ENABLE_SYSMAN` from qmcpack but none of them worked because the L0 runtime gets initialized before main starts.